web/src/components/StateDashboard.js

```diff
@@ -0,0 +1,171 @@
+      >
+        <td>(ex.) Contract</td>
+        <td>Awaiting state response</td>
+        <td>ASAP</td>
+        <td>
+          <Btn size="small" extraCss="col-12 p1 bg-blue white">
+            Response <Icon icon={faArrowRight} className="ml1" />
+          </Btn>
+        </td>
+      </tr>
+      <tr
+        className="align-middle bg-grey-light border-top border-bottom border-white"
+        style={{ borderWidth: '0.5rem' }}
+      >
+        <td>(ex.) APD</td>
+        <td>Submission in progress</td>
+        <td>08/01/2018</td>
+        <td>
+          <Btn size="small" extraCss="col-12 p1 bg-blue white">
+            Open <Icon icon={faEdit} className="ml1" />
+          </Btn>
+        </td>
+      </tr>
+      <tr
+        className="align-middle bg-grey-light border-top border-bottom border-white"
+        style={{ borderWidth: '0.5rem' }}
+      >
+        <td>(ex.) Contract</td>
+        <td>Submission not started</td>
+        <td>12/01/2018</td>
+        <td>
+          <Btn size="small" extraCss="col-12 p1 bg-blue white">
+            Start <Icon icon={faPlusCircle} className="ml1" />
+          </Btn>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+);
+
+const DashboardSection = ({ title, children }) => (
+  <div id="dashboard-tasks" className="mb3 bg-white rounded shadow accordian">
+    <h1 className="col-12 py2 sm-px3 h3 sm-h2 line-height-1 left-align regular blue">
+      {title}
+    </h1>
+    <div className="p2 sm-p3 has-sticky-header">{children}</div>
+  </div>
+);
+DashboardSection.propTypes = {
+  title: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired
+};
+
+const StateDashboard = () => (
+  <div className="site-body">
+    <Sidebar />
+    <div className="site-main p2 sm-p4 md-px0">
+      <TopBtns hideDashboard />
+
+      <SectionTitle>My Dashboard</SectionTitle>
+
+      <DashboardSection title="My Tasks">
+        <TaskTable />
+      </DashboardSection>
+
+      <DashboardSection title="Current Statuses">
+        <CurrentDocuments />
+      </DashboardSection>
+
+      <div className="sm-flex mxn2">
+        <div className="sm-col-6 px2">
+          <DashboardSection title="Recent Activity">
+            <ActivityEntry />
+            <ActivityEntry />
+            <ActivityEntry />
+          </DashboardSection>
+        </div>
+        <div className="sm-col-6 px2">
+          <DashboardSection title="Upcoming events">
+            <h3 className="mt0 pb1 light border-bottom">2018</h3>
+            <Events
+              month="June"
+              events={[
+                { day: 26, title: 'Quarterly Report Due' },
+                { day: 30, title: 'Performance Progress CoP' }
+              ]}
+            />
+            <Events
+              month="July"
+              events={[{ day: 7, title: 'Annual APD Due' }]}
+            />
+          </DashboardSection>
+        </div>
+      </div>
+    </div>
+  </div>
+);
+
+export default StateDashboard;
```

Comment:
Github says this is a new file, but this is just a renamed file with a bunch of changes, so...  I guess that's about the same?

Several of the components in here are essentially placeholders - they're not hooked up to any real data, but we want users to be able to see what the intent is, so we went ahead and built them out.  They're not necessarily super great (especially several components in one file 😬), but I expect they'll get split out over time as they get hooked up to real data.